.github/workflows/slack-noti.yml

```
@@ -11,8 +11,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
       - name: Fire Notification
-        uses: Lubycon/github-reviewer-slack-noti-action@v1.2.1
+        uses: Lubycon/github-reviewer-slack-noti-action@v1.6.0
```

```suggestion
        uses: Lubycon/github-reviewer-slack-noti-action@v2.0.1
```
이왕 해주시는 김에 최신 버전인 `2.0.1`로 설치해주시면 넘나 좋을 것 같습니다 🙏 
`2.0.1`로 올리면 `with`에 있는 `slack-bot-token`이랑 `channel-id`도 제거할 수 있어요!

---

반영했습니당~~ https://github.com/Lubycon/DEVTI-front/pull/84/commits/3499fc670ce180b64bbe42d34b6e4380d149c0e5